app/views/devise/sessions/new.html.erb

```diff
@@ -39,7 +39,9 @@
           <% end -%>
           <div class="form-group actions">
             <%= f.submit "Log in", class: 'btn btn-block btn-primary float-right' %>
-            <button class='btn btn-block btn-secondary float-right' id="oauth">Sign in using OAuth</button>
+            <a class='btn btn-block btn-secondary float-right' href='<%= omniauth_authorize_path(:user, :hpi) %>'>
+              Sign in with HPI OpenID Connect
+            </a>
```

Comment:
is the styling compliance with the style guides yet?

Reply:
i think so, it looks the same like the old button